gov/agendas/deployments.go

```diff
@@ -176,7 +176,7 @@ func agendasForVoteVersion(ver uint32, client DeploymentSource) ([]AgendaTagged,
 // data contents but chaincfg.Vote does not contain the important vote status
 // field that is found in chainjson.Agenda.
 func (db *AgendaDB) updatedb(activeVersions map[uint32][]chaincfg.ConsensusDeployment) (int, error) {
-	var agendas []AgendaTagged
+	agendas := make([]AgendaTagged, 0, len(activeVersions))
```

Comment:
Doesn't hurt here either, but it might be a bit misleading because each `activeVersions` element can result in more than one new `AgendaTagged` being appended (`agendas = append(agendas, taggedAgendas...)`).